.github/labeler.yml

```
@@ -0,0 +1,46 @@
+"area:help centre":
+  - changed-files:
+      - any-glob-to-any-file:
+          - wiki/Help_centre/**
```

doesnt really matter but i think this is supposed to indent more like
```suggestion
"area:help centre":
  - changed-files:
    - any-glob-to-any-file:
      - wiki/Help_centre/**
```
(and all the others)

---

My formatters (Prettier and Red Hat) say that it is formatted like this, I searched and [this page (of prettier)](https://prettier.io/docs/en/configuration.html#configuration-overrides) is an example of this